src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int FIRST_HALF_ARRAY = 0;
+    private static final int SECOND_HALF_ARRAY = 1;
+    private static final int SIZE_REPORT = 3;
 
+    public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
It is necessary to break the logic of the method into 3 separate private methods:
1. Reading data from a file
2. Create a report
3. Writing a report to a file

The getStatistic() method should only have calls to these three methods